docs/develop/python/message-passing.mdx

```diff
@@ -405,14 +405,37 @@ Sometimes, `async def` Signal or Update handlers need to meet certain conditions
 You can use a wait condition ([`workflow.wait_condition`](https://python.temporal.io/temporalio.workflow.html#wait_condition)) to set a function that prevents the code from proceeding until the condition returns `True`.
 This is an important feature that helps you control your handler logic.
 
-Here are two important use cases for `workflow.wait_condition`:
+Here are three important use cases for `workflow.wait_condition`:
 
-- Waiting in a handler until it is appropriate to continue.
-- Waiting in the main Workflow until all active handlers have finished.
+- Wait for a Signal or Update to arrive.
```

Comment:
```suggestion
- Wait for a Signal or Update to arrive. You might be waiting on an external process or person to sign off and approve the next steps.
```

Reply:
I've tried to put in sample wording for these three points, but obviously I'm not a subject matter expert. They're examples of the _kinds_ of wording that would strengthen these points.